ui/src/pages/sitemap.xml.tsx

```diff
@@ -0,0 +1,45 @@
+import * as Types from '@types';
+import * as api from '@api';
+import * as Config from '@config';
+
+const PublicationSitemapIndex = () => {
```

Comment:
The name should be Sitemap. This should return the sitemap index with all our sitemaps located under /sitemaps

Under /sitemaps folder we do all the fetching and filtering if needed. Each resource that needs to be fetched in order to generate a sitemap index should be located under /sitemaps for example:

`/sitemaps
            /static.xml.tsx
            /publications.xml.tsx
            /topics.xml.tsx
            /users.xml.tsx
            /blogs.xml.tsx`

Reply:
I've split these out into seperate files in the latest commit; they each export a var called Sitemap